packages/sdk-components-react/src/components.ts

```diff
@@ -32,3 +32,5 @@ export { VimeoPlayButton } from "./vimeo-play-button";
 export { VimeoSpinner } from "./vimeo-spinner";
 export { XmlNode } from "./xml-node";
 export { Time } from "./time";
+export { Select } from "./select";
+export { Option } from "./option";
```

Comment:
Do we need them separate? Or could be grouped like radix components?

Reply:
Props are not generated in a single file

Reply:
How so?
<img width="574" alt="image" src="https://github.com/webstudio-is/webstudio/assets/5635476/715fad1a-95d1-4d95-8d5a-3b0ac1c2f375">